core/src/main/java/org/eclipse/hono/util/CommandRouterConstants.java

```diff
@@ -0,0 +1,94 @@
+/*******************************************************************************
+ * Copyright (c) 2020 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+
+package org.eclipse.hono.util;
+
+/**
+ * Constants &amp; utility methods used throughout the Command Router API.
+ */
+
+public final class CommandRouterConstants extends RequestResponseApiConstants {
+
+    /**
+     * The name of the Command Router API endpoint.
+     */
+    public static final String COMMAND_ROUTER_ENDPOINT = "cmd_router";
+
+    /**
+     * Request actions that belong to the Command Router API.
+     */
+    public enum CommandRouterAction {
+        /**
+         * The <em>set last known gateway for device</em> operation.
+         */
+        SET_LAST_KNOWN_GATEWAY("set-last-known-gw"),
+        /**
+         * The <em>register command consumer</em> operation.
+         */
+        REGISTER_COMMAND_CONSUMER("register-command-consumer"),
+        /**
+         * The <em>unregister command consumer</em> operation.
+         */
+        UNREGISTER_COMMAND_CONSUMER("unregister-command-consumer"),
+        /**
+         * The <em>unknown</em> operation.
+         */
+        UNKNOWN("unknown");
```

Comment:
do we need this?

Reply:
I've followed the same approach here as with the corresponding classes from the DeviceConnection, Tenant and Credentials APIs. It's used so that `CommandRouterAction.from` doesn't return `null`, meaning no extra null check is needed to use the return value in a "switch .. case".